cluster-provision/okd/4.1/provision.sh

```diff
@@ -23,5 +23,5 @@ ${gocli} provision okd \
 --master-memory 10240 \
 --installer-pull-token-file ${INSTALLER_PULL_SECRET} \
 --installer-repo-tag release-4.1 \
---installer-release-image docker.io/kubevirtci/ocp-release:4.1.18 \
+--installer-release-image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:30857847ade065a57e02c9297cafdd6b2a95d086d0d99480980a3a6eaba2a95b \
```

Comment:
why does it appear as v4.0? also, take into account that our release image contains some libvirt fixes

Reply:
Also trying to install it locally I get this

```
57e02c9297cafdd6b2a95d086d0d99480980a3a6eaba2a95b, name=cocky_hugle)
Oct 23 10:09:51 test1-87jxt-bootstrap podman[3924]: 2019-10-23 10:09:51.084608002 +0000 UTC m=+0.348919177 container remove de5b60c9d9a282be0f7805c146f1b5b0ef3aebb67858325e3c7cab2f25bdbba8 (image=quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:30857847ade065a57e02c9297cafdd6b2a95d086d0d99480980a3a6eaba2a95b, name=cocky_hugle)
Oct 23 10:09:51 test1-87jxt-bootstrap bootkube.sh[3910]: Error: container_linux.go:345: starting container process caused "exec: \"/noentry\": stat /noentry: no such file or directory"
Oct 23 10:09:51 test1-87jxt-bootstrap bootkube.sh[3910]: : OCI runtime error
Oct 23 10:09:51 test1-87jxt-bootstrap systemd[1]: bootkube.service: Main process exited, code=exited, status=127/n/a
Oct 23 10:09:51 test1-87jxt-bootstrap systemd[1]: bootkube.service: Failed with result 'exit-code'.
Oct 23 10:09:56 test1-87jxt-bootstrap systemd[1]: bootkube.service: Service RestartSec=5s expired, scheduling restart.
Oct 23 10:09:56 test1-87jxt-bootstrap systemd[1]: bootkube.service: Scheduled restart job, restart counter is at 24.
Oct 23 10:09:56 test1-87jxt-bootstrap systemd[1]: Stopped Bootstrap a Kubernetes cluster.
Oct 23 10:10:06 test1-87jxt-bootstrap systemd[1]: Started Bootstrap a Kubernetes cluster.
Oct 23 10:10:07 test1-87jxt-bootstrap podman[4020]: 2019-10-23 10:10:07.169043317 +0000 UTC m=+0.075651922 container create 1d332134cb671f068937a8f006787ead07d18e9151cf794d25cbfccbd390006d (image=quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:30857847ade065a57
```